big_scape/network/network.py

```diff
@@ -96,7 +96,7 @@ def get_edge(
     if edge is None:
         return None
 
-    return cast(tuple[int, int, float, float, float, float, str], edge)
+    return cast(tuple[int, int, float, float, float, float, str], edge[0:7])
```

Comment:
While you're here, can you refactor this a bit to select columns in the select_statement on 86 instead of getting everything and returning a slice of the result. e.g.

```python
distance_table = DB.metadata.tables["distance"]
select_columns = (
  distance_table.c.region_a_id,
  distance_table.c.region_b_id, # note the last comma; this is a tuple
)

select_statement = (
   distance_table
   .select(columns)
```

something like that. not essential if you want to skip it.